### PR TITLE
[Visualizer] Create React App for visualizer

### PR DESCRIPTION
**Pull request by parkjisu6239**
React will be used for visualization. These files are basically created when create a React app.
ONE-vscode-DCO-1.0-Signed-off-by: parkjisu6239 parkjisu6239@gmail.com

### CODE REVIEW

**Comment by seanshpark** [comment]
I've used `[Project]` prefix to add files in `Project` folder and related.
I think you should use another prefix if this is not your intention.

**Comment by seanshpark** [comment]
Please split files like `.gitignore` and others, by your purpose

File: src/Visualizer/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (seanshpark) [inline src/Visualizer/.eslintrc.json:1]: same question as https://github.com/Samsung/ONE-vscode/pull/93#discussion_r709817145

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,24 @@ out
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (seanshpark) [inline .gitignore:25]: Please remoe this red circle image

image

You need to add an empty line at the end

**Comment by seanshpark** [comment]
Read this again carefully

https://github.com/Samsung/ONE-vscode/issues/87#issuecomment-919753850

These are by thoughts about PRs
- one context per PR, that is, one purpose of objective in one PR; that is there can be lots of PRs; this will take time to land a huge change
- too many purpose makes understanding codes difficult
- one who wrote the code knows what one is doing but others' don't
- communicating by written word is very hard

**Comment by parkjisu6239** [comment]
@seanshpark

The files I uploaded now are default files created when I create an application with React framework.
It's not a folder that we created for random purposes.
So I thought it was unnecessary to subdivide it.
[React docs](https://ko.reactjs.org/docs/create-a-new-react-app.html)

Should we divide these into different PRs?
There are no folders or files like this in the PR created later.

File: .gitignore (hunk 2 above)
Comment (parkjisu6239) [reply .gitignore]: thanks your feedback!

**Comment by seanshpark** [comment]
> React docs
This file is a structure, so it doesn't work without any.

Please check like #93 

> hould we divide these into different PRs?

Yes

> There are no folders or files like this in the PR created later.

I don't understand what you mean

File: src/Visualizer/.eslintrc.json (hunk 1 above)
Comment (wnsdud4197) [reply src/Visualizer/.eslintrc.json:1]: When eslint encounters `package.json` or `.esconfig.json` during the `npm run lint`, the nearest file is overriding. There is a problem with the eslintconfig set in the `Visualizer/package.json` , so we will remove the setting and remove the `Visualizer/.eslintrc.json`

Thank you for the feedback! 🙏

**Comment by parkjisu6239** [comment]
I'll subdivide it and upload it again! I'll close this PR

**Pull request closed without merging**